### PR TITLE
test_loggerd: make rotation test more robust

### PR DESCRIPTION
**Pull request by jnewb1**
seems like the randomness caused occasional failures

### CODE REVIEW

File: system/loggerd/tests/test_loggerd.py [hunk 1]
```diff
@@ -152,8 +152,8 @@ def test_rotation(self):
       vipc_server.create_buffers_with_sizes(stream_type, 40, False, *(frame_spec))
     vipc_server.start_listener()
 
-    num_segs = random.randint(2, 5)
-    length = random.randint(1, 3)
+    num_segs = 3
```
Comment (adeebshihadeh) [inline system/loggerd/tests/test_loggerd.py:155]: why is it more robust?

Comment (jnewb1) [reply system/loggerd/tests/test_loggerd.py:155]: seems like some combination of these random values causes issues in CI, I got this error a bunch: https://github.com/commaai/openpilot/actions/runs/7646074551/job/20834139473#step:6:176

Comment (jnewb1) [reply system/loggerd/tests/test_loggerd.py:155]: https://github.com/commaai/openpilot/actions/runs/7660038182/job/20876481578?pr=31161#step:6:140

Comment (jnewb1) [reply system/loggerd/tests/test_loggerd.py:155]: I also couldn't replicate the issue locally with any combination

**Pull request merged**